src/main.tsx

```diff
@@ -2,30 +2,30 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.tsx';
 import { ConfigProvider } from 'antd';
+import './global.scss';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
     <React.StrictMode>
         <ConfigProvider
             theme={{
                 components: {
                     Collapse: {
-                        headerBg: '#95223F',
+                        headerBg: '#C6264E',
```

Comment:
We should create a colors.scss file and export css variables from there like `headerColor` etc. and then use those variables instead of hardcoded values. But this could also be addressed in a future Ticket/Pull Request

Reply:
Fixed in Future Commit